From 1352f5020bd0f7aa063a1d329c52a688f12eb602 Mon Sep 17 00:00:00 2001
From: Bryan O'Donoghue <bryan.odonoghue@linaro.org>
Date: Fri, 6 Sep 2019 11:36:07 +0100
Subject: [PATCH 09/13] pico-imx6ul_bl33: configs: Enable CONFIG_OF_LIBFDT

In order to switch on DTB overlay support in WaRP7 BL33 we first need to
switch on LIBFDT support. Do that now.

Signed-off-by: Bryan O'Donoghue <bryan.odonoghue@linaro.org>
---
 configs/pico-imx6ul_bl33_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/pico-imx6ul_bl33_defconfig b/configs/pico-imx6ul_bl33_defconfig
index bd1e075a90..ae059debaa 100644
--- a/configs/pico-imx6ul_bl33_defconfig
+++ b/configs/pico-imx6ul_bl33_defconfig
@@ -64,3 +64,4 @@ CONFIG_OPTEE_SKIP_LOWLEVEL_INIT=y
 CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
 CONFIG_FIT_VERBOSE=y
+CONFIG_OF_LIBFDT=y
-- 
2.22.0

